File: src/main/java/core/data/DataTable.java
package core.data;

import java.util.ArrayList;
import java.util.Dictionary;
import java.util.HashSet;

//TODO: (avlomakin) find java DataTable class
public class DataTable {

    //temporary public
    public HashSet<String> columns;
    public ArrayList<Dictionary<String, String>> rows;

    public DataTable() {
        columns = new HashSet<>();
        rows = new ArrayList<>();
    }

    public void addColumn(String name)
    {
        columns.add(name);
    }

    public void addRow(Dictionary<String, String> row)
    {
        //TODO: (avlomakin) add row validation
        rows.add(row);
    }
}

File: src/main/kotlin/com/mirf/core/data/DataTable.kt
package com.mirf.core.data

import java.util.ArrayList
import java.util.Dictionary
import java.util.HashSet

//TODO: (avlomakin) find java DataTable class
class DataTable(var columns: HashSet<String> = HashSet(), var rows: ArrayList<HashMap<String, String>> = ArrayList()) {

    fun addColumn(name: String) {
        columns.add(name)
    }

    fun addRow(row: HashMap<String, String>) {
        //TODO: (avlomakin) add row validation
        rows.add(row)
    }
}
